### PR TITLE
Trivial cleanups

### PR DESCRIPTION
**Pull request by keszybz**
Stuff I spotted when going through the commits when preparing backports.

### CODE REVIEW

File: src/test/test-bootspec.c [hunk 1]
```diff
@@ -8,7 +8,6 @@
 #include "tmpfile-util.h"
 
 TEST_RET(bootspec_sort) {
-
         static const struct {
```
Comment (YHNdnzj) [inline src/test/test-bootspec.c]: Uh, I actually prefer that we add a newline here...

Comment (keszybz) [reply src/test/test-bootspec.c]: We don't put an empty line between a one-line function signature and the variable declaration block.

Comment (YHNdnzj) [reply src/test/test-bootspec.c]: We often put `static const` structs/arrays at the beginning of the function, and insert a newline regardless of function parameters. I think that way is nicer, but feel free to disagree and merge this.

Comment (keszybz) [reply src/test/test-bootspec.c]: I dropped that last commit. This was a trivial change, and it doesn't seem worth to do it if there is no universal agreement.

**Pull request merged**